CHANGELOG.md

```diff
@@ -0,0 +1,207 @@
+# Changelog
+
+## v1.11.0-rc1
+
+Summary of Changes
+------------------
+
+**Major Changes:**
+* Enable CiliumEndpointSlice feature (#17658, @krishgobinath)
+
+**Minor Changes:**
+* Add flag to list all available configurations (#17303, @h3llix)
+* Add WireGuard status to cilium encrypt. (#17684, @h3llix)
+* crd: Add categories for cilium CRDs (#17162, @sayboras)
+* datapath,daemon: Enable multi-dev XDP (#17655, @brb)
+* datapath: Add a flag to set VXLAN and Geneve ports (#16874, @errordeveloper)
+* Detect devices from global unicast routes in addition to only
 looking for the device with the Kubernetes Node IP and the one with
 default route.  This expands the set of devices used for kube-proxy
 replacement, host firewall and bandwidth manager and should reduce
 the need to specify devices manually. (#17219, @joamaki)
```

Comment:
Should the extra `^M` be removed?

Reply:
CC @joamaki this is how newlines show up in the release notes, typically we end up having to strip them out or mess with the release notes to get them to render right. Shorter more concise release notes can mitigate this.